fdm-app/.env.example

```
@@ -162,6 +162,15 @@ SENTRY_AUTH_TOKEN=
 PUBLIC_POSTHOG_KEY= # Example: phc_...
 PUBLIC_POSTHOG_HOST= # Example: https://eu.i.posthog.com
 
+# PostHog Query API credentials for server-side usage tracking (e.g. Gerrit daily rate limit).
+# Required only if per-user daily rate limiting for Gerrit is desired.
+# POSTHOG_PROJECT_ID: numeric project ID (find it in PostHog → Project Settings → Project ID).
+# POSTHOG_PERSONAL_API_KEY: a personal API key with Query read access (phx_...).
+#   Create one at: https://app.posthog.com/settings/user-api-keys
+# When either variable is missing the rate limit check is skipped (fail-open).
+POSTHOG_PROJECT_ID= # Example: 12345
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)